src/Core/Console/DocBloxErrorChecker.php

```diff
@@ -59,7 +59,6 @@ protected function doExecute()
 			throw new \RuntimeException('DocBlox isn\'t available.');
 		}
 
-		//return from util folder to frindica base dir
```

Comment:
Why removing the comment when the command to change the directory is still needed? I guess it is needed now for other reasons?

Reply:
Because it isn't a `cd` command, we just use the Friendica directory as a base directory in a variable.